logstash-core/lib/logstash/pipeline.rb

```diff
@@ -156,15 +142,15 @@ def start_workers
       @filters.each {|f| f.register}
 
       pipeline_workers = safe_pipeline_worker_count
-      batch_size = @settings[:pipeline_batch_size]
-      batch_delay = @settings[:pipeline_batch_delay]
+      batch_size = @settings["pipeline.batch.size"]
+      batch_delay = @settings["pipeline.batch.delay"]
       max_inflight = batch_size * pipeline_workers
       @logger.info("Starting pipeline",
-                   :id => self.pipeline_id,
-                   :pipeline_workers => pipeline_workers,
-                   :batch_size => batch_size,
-                   :batch_delay => batch_delay,
-                   :max_inflight => max_inflight)
+                   "id" => self.pipeline_id,
```

Comment:
I generally agree with @purbon here. It'd be nice to have a policy, but until we finish the mythical style guide I'm inclined to just let the author pick. Strings are fine with me (Though `:"foo.bar"` is a valid symbol).